tanagra-aou-utils/tanagra_env_variables_template.yaml

```diff
@@ -0,0 +1,14 @@
+env_variables:
+  TANAGRA_DB_CLOUD_SQL_INSTANCE: $CLOUD_SQL_INSTANCE_NAME
+  TANAGRA_DB_DRIVER_CLASS_NAME: com.mysql.cj.jdbc.Driver
+  TANAGRA_DB_PASSWORD: $WORKBENCH_DB_PASSWORD
+  TANAGRA_DB_SOCKET_FACTORY: com.google.cloud.sql.mysql.SocketFactory
+  TANAGRA_DB_URI: jdbc:mysql:///tanagra_db
+  TANAGRA_DB_USERNAME: $WORKBENCH_DB_USER
+  TANAGRA_ACCESS_CONTROL_BASE_PATH: https://$TANAGRA_ACCESS_CONTROL_BASE_PATH
+  TANAGRA_ACCESS_CONTROL_MODEL: $TANAGRA_ACCESS_CONTROL_MODEL
+  TANAGRA_AUTH_GCP_PROJECT_ID: $TANAGRA_AUTH_GCP_PROJECT_ID
+  TANAGRA_UNDERLAY_FILES: $TANAGRA_UNDERLAY_FILES
+  TANAGRA_AUTH_DISABLE_CHECKS: $TANAGRA_AUTH_DISABLE_CHECKS
+  TANAGRA_AUTH_BEARER_TOKEN: $TANAGRA_AUTH_BEARER_TOKEN
+  TANAGRA_AUTH_IAP_GKE_JWT: false
```

Comment:
why do I need this?
TANAGRA_AUTH_IAP_GKE_JWT -> is this setting those 3-auth types to false?
TANAGRA_AUTH_GCP_PROJECT_ID

Reply:
API starts with a null pointer exception if this is left out. I'm going to bring it up with Mariko